pkgs/applications/misc/pgmanage/default.nix

```diff
@@ -11,6 +11,10 @@ stdenv.mkDerivation rec {
     sha256 = "0g9kvhs9b6kc1s7j90fqv71amiy9v0w5p906yfvl0j7pf3ayq35a";
   };
 
+  patchPhase = ''
+    sed -e "s~#!/bin/sh~#!$SHELL ~" -i src/configure
```

Comment:
patchShebangs maybe?

Reply:
I did not know of patch Shebangs, I’ll look into it. Based on its name, it should do exactly what I expect it to do.